http/analysis-service/server.py

```diff
@@ -126,15 +107,23 @@ def update_job(job):
     update = table.update().where(table.c.id == job['id']).values(job)
     return db['conn'].execute(update)
 
-def get_jobs(owner=None):
+def get_jobs(owner=None, is_cluster=None):
     db = get_db()
     table = db['metadata'].tables['scheduled_jobs']
+
+    if is_cluster is None:
+        condition = True
+    elif is_cluster:
+        condition = (table.c.code_uri.endswith(".ipynb"))
+    else:
+        condition = (table.c.code_uri.endswith(".tar.gz"))
```

Comment:
It might be better to have a table field to determine whether a job should be treated as a "cluster" job or not. Alternatively, the `worker_count` field might be a more reliable indicator. At the least, ".tgz" should also be recognized as a valid non-cluster code uri suffix.